robots/commenter/main.go

```diff
@@ -190,11 +207,23 @@ func main() {
 		asc = true
 	}
 	commenter := makeCommenter(o.comment, o.useTemplate)
-	if err := run(c, query, sort, asc, o.random, commenter, o.ceiling); err != nil {
+	if err := run(ctx, c, githubClient, query, sort, asc, o.random, commenter, o.ceiling, o.labelsUpdated); err != nil {
 		log.Fatalf("Failed run: %v", err)
 	}
 }
 
+func createGithubClient(ctx context.Context, endpoint flagutil.Strings, tokenPath string) (*github2.Client, error) {
+	log.Print("Create Github client")
+	tokenByte, err := ioutil.ReadFile(tokenPath)
+	if err != nil {
+		log.Fatalf("Failed reading token: %v", err)
```

Comment:
let's return error here. Usually funcs don't just kill the current process (also we already fail on error one layer above)

Reply:
Damn, sure. Of course